src/blockchain/p2p/peer/Address.cpp

```diff
@@ -39,8 +42,7 @@ auto Address::Chain() const noexcept -> blockchain::Type
 auto Address::Display() const noexcept -> UnallocatedCString
 {
     Lock lock(lock_);
-
-    return address_->Display();
+    return address_ ? address_->Display() : std::string("BOTCHED");
```

Comment:
What is Botched ? maybe we can add some static member to provide this string value ?

Reply:
Second remark - BOTCHED path should notify us of an erroneous condition; method is noexcept, but is returning that string strong enough to report the problem?

Reply:
I'll remove this testing leftover. When the problem occurred, it was massive, this was just the firs place it struck when running one particular test. It is one of the cases where the problem just have to be prevented rather than detected.